lib/weighable/model.rb

```diff
@@ -13,6 +13,7 @@ def weighable(column, presence: false, store_as: :gram, precision: nil)
 
       def define_setter(column, store_as: :gram, precision: nil)
         define_method "#{column}=" do |weight|
+          weight = weight.with_indifferent_access if weight.respond_to?(:with_indifferent_access)
```

Comment:
`with_indifferent_access` comes from ActiveSupport. Since this is a public gem, it might be used by people not having ActiveSupport in their dependencies. I'd rather check if weight is a hash and e.g. stringify keys manually. Also, I think a spec for it would be welcome.

Reply:
Mi scusi, we have activesupport https://github.com/greenbits/weighable/blob/master/weighable.gemspec#L23 🤦‍♂️